impeller/tessellator/tessellator.h

```diff
@@ -44,10 +44,17 @@ class Tessellator {
 
   ~Tessellator();
 
+  /// @brief A callback that returns the results of the tessellation.
+  ///
+  ///        The index buffer may not be populated, in which case [indices] will
+  ///        be nullptr. The [vertices_size] is the size of the vertices array
+  ///        and not the number of vertices, which is usually vertices_size / 2.
+  ///        In cases where the indices is nullptr, vertex_or_index_count will
+  ///        contain the count of indices.
   using BuilderCallback = std::function<bool(const float* vertices,
                                              size_t vertices_size,
                                              const uint16_t* indices,
-                                             size_t indices_size)>;
+                                             size_t vertex_or_index_count)>;
```

Comment:
Overloading this variable doesn't seem prudent.  Can't it just be indices_size and be zero if `indices == nullptr`?  We didn't need vertex count in the past

Reply:
The reason this is overloaded is a bit of an issue yes, I should just fix it. Really the problem is that these variables aren't counts, they are the value to multiply with sizeof(float)/sizeof(uint16_t) to compute offsets. But I just make them counts, then this problem goes away.

Reply:
Done